CSS/module.css

```diff
@@ -0,0 +1,154 @@
+/*reusable styles*/
+
+.button {
+  border: none;
+  font-family: Open Sans Condensed;
+  font-size: 15px;
+}
+
+.paragraphText {
+  font-family: Scheherazade;
+  font-size: 24px;
+  color: #5D5D5D;
+}
+
+.leftFloat {
+  float: left;
+  margin-left: 1em;
+}
+
+.rightFloat {
+  float: right;
+}
+
+.textGray {
```

Comment:
This isn't the best way to use classes, It's basically the same as doing this in html: `style="color: #BBBABA"`. Not very maintainable.